modules/core/src/main/java/org/apache/ignite/internal/processors/query/GridQueryFinishedInfo.java

```diff
@@ -45,12 +46,28 @@ public class GridQueryFinishedInfo {
     /** */
     private final long finishTime;
 
+    /** Query cancellable flag. */
```

Comment:
what reason to have the field?

Reply:
Looks like you're right. Deleted this field.